packages/account/src/Components/api-token/__tests__/api-token-table.spec.tsx

```diff
@@ -21,9 +17,21 @@ describe('ApiTokenTable', () => {
             },
         ],
         deleteToken: jest.fn(),
-        footer_ref: document.createElement('div'),
-        overlay_ref: document.createElement('div'),
-        toggleOverlay: jest.fn(),
+    };
+    const store = mockStore({
+        ui: {
+            is_mobile: false,
+        },
+    });
```

Comment:
```suggestion
    const store = mockStore({ });
```
it is already false by default. we can omit it here